### ISSUE TITLE
No module named 'packaging'

### ISSUE THREAD

**Issue by guigui64** [ofirgall/tmux-window-name#23]
Hi,

I just installed this plugin but it does not work on my setup.
I tried running the script manually and got this error: `No module named 'packaging'`

I am using python 3.10.6

Let me know if you need some more information.

**Comment by ofirgall** [comment]
Sorry for the late response, can you share the full output of the script execution?

**Comment by SilverRainZ** [comment]
- Commit: master ([a10a0d7](https://github.com/ofirgall/tmux-window-name/commit/a10a0d7d9ff1eaafdfd8054a1af0b53b6016d75a))
- Python 3.10.9
- libtmux 0.16.0 (due to #20)
- Full output:

```
 Traceback (most recent call last):
  File "/home/xxx/.config/tmux/plugins//tmux-window-name/scripts/rename_session_windows.py", line 12, in <module>
    import libtmux
  File "/home/xxx/.local/lib/python3.10/site-packages/libtmux/__init__.py", line 12, in <module>
    from .pane import Pane
  File "/home/xxx/.local/lib/python3.10/site-packages/libtmux/pane.py", line 12, in <module>
    from libtmux.common import tmux_cmd
  File "/home/xxx/.local/lib/python3.10/site-packages/libtmux/common.py", line 17, in <module>
    from ._compat import LooseVersion, console_to_str, str_from_console
  File "/home/xxx/.local/lib/python3.10/site-packages/libtmux/_compat.py", line 40, in <module>
    from packaging.version import Version
ModuleNotFoundError: No module named 'packaging'
```

**Comment by Joshswooft** [comment]
Also having this issue

**Comment by newellista** [comment]
I'm having the same issue. However, I'm running on OSX 13.1
Python 3.10.9
libtmux 0.16.0

```
Traceback (most recent call last):                                                                                               15:19:24
  File "/Users/stevenewell/.tmux/plugins//tmux-window-name/scripts/rename_session_windows.py", line 12, in <module>
    import libtmux
  File "/Users/stevenewell/Library/Python/3.10/lib/python/site-packages/libtmux/__init__.py", line 12, in <module>
    from .pane import Pane
  File "/Users/stevenewell/Library/Python/3.10/lib/python/site-packages/libtmux/pane.py", line 12, in <module>
    from libtmux.common import tmux_cmd
  File "/Users/stevenewell/Library/Python/3.10/lib/python/site-packages/libtmux/common.py", line 17, in <module>
    from ._compat import LooseVersion, console_to_str, str_from_console
  File "/Users/stevenewell/Library/Python/3.10/lib/python/site-packages/libtmux/_compat.py", line 40, in <module>
    from packaging.version import Version
ModuleNotFoundError: No module named 'packaging'
```

**Comment by SilverRainZ** [comment]
This is an upstream issue and already fixed in libtmux v0.16.1, see https://github.com/tmux-python/libtmux/issues/460

**Comment by ofirgall** [comment]
@SilverRainZ thanks for investigating the issue.

Changed the README to suggest installing libtmux 0.16.1
